### DISCUSSION TITLE
How to change base branch of git gutter for all buffers?

### DISCUSSION THREAD

**Discussion by wh75er** [helix-editor/helix#9128] [Q&A]
e.g. in vim GitSigns plugin you could type GitSigns change_base master true (enable for all buffers)

Is it possible to do in helix? How? I would like to have diff from my current git/feature_branch with git/master branch

**Comment by the-mikedavis** [comment]
This isn't possible currently, see https://github.com/helix-editor/helix/issues/5825

**Reply by pascalkuthe** [reply]
Most of the infrastructure for that feature is implemented in #8056
